playground/backend/cmd/server/controller.go

```diff
@@ -272,6 +282,9 @@ func processCode(ctx context.Context, cacheService cache.Cache, lc *fs_tool.Life
 	errorChannel := make(chan error, 1)
 	dataChannel := make(chan interface{}, 1)
 	successChannel := make(chan bool, 1)
+	cancelChan := make(chan bool, 1)
```

Comment:
```suggestion
	cancelChannel := make(chan bool, 1)
```

Let's keep consistent naming for channel variables

Reply:
Renamed.